apps/demo/src/app/pages/notification/examples/03-close-notification/index.ts

```diff
@@ -0,0 +1,37 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {NotificationService} from '@ng-web-apis/notification';
+import {PermissionsService} from '@ng-web-apis/permissions';
+import {timer} from 'rxjs';
+import {filter, switchMap, takeUntil} from 'rxjs/operators';
+
+@Component({
+    selector: 'notification-page-example-3',
+    templateUrl: './index.html',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class NotificationPageExample3 {
+    readonly notificationPermissionState$ =
+        this.permissionsService.state('notifications');
+
+    constructor(
+        private readonly notificationService: NotificationService,
+        private readonly permissionsService: PermissionsService,
+    ) {}
+
+    sendNotification(): void {
+        this.notificationService
+            .requestPermission()
+            .pipe(
+                filter(permission => permission === 'granted'),
+                switchMap(() =>
+                    this.notificationService.open('Close me, please!', {
```

Comment:
How about we rename services to `permissions` and `notifications` in every example? Would probably look more concise.

Reply:
✅